### PR TITLE
114 the grpc framework supports non streaming methods getlatestblockhash

### PR DESCRIPTION
**Pull request by shuimuliang**

### CODE REVIEW

File: yellowstone-grpc-geyser/src/grpc.rs [hunk 1]
```diff
@@ -25,6 +26,7 @@ use {
     std::{
         collections::HashMap,
         sync::atomic::{AtomicUsize, Ordering},
+        sync::{Arc, RwLock},
```
Comment (fanatid) [inline yellowstone-grpc-geyser/src/grpc.rs]: I think it's better to use RwLock from tokio, everything what we have is async

Comment (shuimuliang) [reply yellowstone-grpc-geyser/src/grpc.rs:29]: notify_block_metadata is not async fn, if call await in it
```rust
    if let Ok(mut latest_block_meta_write_guard) = inner.latest_block_meta.write().await {
        *latest_block_meta_write_guard = Some(block_meta.clone());
    }
```
error[E0728]: `await` is only allowed inside `async` functions and blocks

**Comment by shuimuliang** [comment]
for your review @fanatid

File: yellowstone-grpc-geyser/src/grpc.rs (hunk 1 above)
Comment (fanatid) [reply yellowstone-grpc-geyser/src/grpc.rs]: oh, I missed that we set latest value in the sync function, so use std RwLock was good idea. Sorry.

Comment (fanatid) [reply yellowstone-grpc-geyser/src/grpc.rs]: https://github.com/rpcpool/yellowstone-grpc/pull/118

**Pull request merged**